scio-core/src/main/scala/com/spotify/scio/hash/ApproxFilter.scala

```diff
@@ -278,12 +278,10 @@ sealed trait ApproxFilterCompanion {
       logger.info(
         "Partition settings for approximate filter side input of {} keys: " +
```

Comment:
So looks like that this doesnt fly with 2.12. It's probly better to use string interpolation here. Also don't forget to apply `scalafmt`